src/functions/Get-SkipRemainingOnFailurePlugin.ps1

```diff
@@ -0,0 +1,110 @@
+function New-SkippedTestMessage {
+    [OutputType([string])]
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [Pester.Test]
+        $Test
+    )
+    "Skipped due to previous failure at path:line '$($Test.ExpandedPath):$($Test.StartLine)' and Run.SkipRemainingOnFailure set to '$($PesterPreference.Run.SkipRemainingOnFailure.Value)'"
```

Comment:
`$Test.ExpandedPath` is DescribeName.ContextName.TestName so ExpandedPath:StartLine looks a bit weird:
`[!] c Skipped due to previous failure at path:line 'a.b myvarvalue:9' and Run.SkipRemainingOnFailure set to 'Block'`

Reply:
Yeah this is a good point.  I originally had it in there to make it easier to find the failing test, but don't think we need it since expanded path shows everything we need. Also makes tests easier to write since I don't need to worry about line numbers 👍